src/types/eatery_type.py

```diff
@@ -12,17 +12,26 @@ class CoordinatesType(ObjectType):
   longitude = Float(required=True)
 
 class EateryType(ObjectType):
-  about = String(required=True)
-  about_short = String(required=True)
-  campus_area = Field(CampusAreaType, required=True)
   coordinates = Field(CoordinatesType, required=True)
   eatery_type = String(required=True)
   id = Int(required=True)
   image_url = String(required=True)
-  location = String(required=True)
   name = String(required=True)
-  name_short = String(required=True)
-  operating_hours = List(OperatingHoursType, required=True)
   payment_methods = Field(PaymentMethodsType, required=True)
   phone = String(required=True)
+
+class CampusEateryType(EateryType):
+  about = String(required=True)
+  about_short = String(required=True)
+  campus_area = Field(CampusAreaType, required=True)
+  location = String(required=True)
+  name_short = String(required=True)
```

Comment:
same thing regarding `name_short`, i understand that the `json` may have it but if it's unused we should toss it